components/Slider/src/MDCSlider.h

```diff
@@ -212,6 +222,16 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign) NSUInteger numberOfDiscreteValues;
 
+/**
+ Configures the visibility of the track tick marks.
+
+ @note Unless this property is set explicitly, its value will change based on the value of
```

Comment:
Nit: This comment makes me think that we should have an additional enum value to let our clients return it to the "default" changing state.

I could see a client wanting to set it back because they are reconstituting a slider from archive or transition animation. However I could see that as a much less common use case and we could build it when they need it.

Reply:
Thanks for the call-out.  I agree that this seems less common than most use cases. I would be surprised that if a "restored" Slider would behave differently than how it was configured originally.